### PR TITLE
Update Web Template from Template repo

### LINKED ISSUE #4830: Update Web Templates to the latest

### PR DESCRIPTION
**Pull request by phenning**
Fixes #4830 

@mlorbetske @piotrpMSFT  @MattGertz @srivatsn can you sign off please?

### CODE REVIEW

File: src/dotnet/commands/dotnet-new/CSharp_Web/$projectName$.csproj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="15.0">
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
 
   <PropertyGroup>
```
Comment (srivatsn) [inline src/dotnet/commands/dotnet-new/CSharp_Web/$projectName$.csproj]: We should remove the Exlcude=$(GlobalExclude) from the Compile and EmbeddedResource includes.  The SDK already has a Remove entry for the excludes and so we don't need it in the project file.

Comment (phenning) [inline src/dotnet/commands/dotnet-new/CSharp_Web/$projectName$.csproj]: Pushed a change with that.

**Comment by mlorbetske** [comment]
:shipit:

**Comment by livarcocc** [comment]
@dotnet-bot Test CentOS7.1 x64 Debug Build

**Comment by livarcocc** [comment]
@srivatsn can you add the approval label?

**Pull request merged**